src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -17,21 +17,24 @@
     /** The application should exit. */
     private final boolean exit;
 
+    private final boolean history;
+
     /**
      * Constructs a {@code CommandResult} with the specified fields.
      */
-    public CommandResult(String feedbackToUser, boolean showHelp, boolean exit) {
+    public CommandResult(String feedbackToUser, boolean showHelp, boolean exit, boolean history) {
```

Comment:
Instead of appending additional booleans to the creation CommandResult, wouldn't making it an ENUM be better? Since all of the UI interactions are caused by disjoint commands, hence making it 1 ENUM would look significantly neater?